lib/provisioner/plugin/automator.rb

```diff
@@ -66,6 +68,20 @@ def runTask
         end
       end
 
+      def verify_ssh_host_key(host, type = 'rsa')
```

Comment:
Why do both verify_ssh_host_key (in this file) and ssh_keyscan (in utils.rb) methods have type = 'rsa' as input parameters if we pass type to ssh_keyscan from verify_ssh_host_key where type is already set to rsa?